src/styles/StyleUtils.ts

```diff
@@ -534,17 +542,17 @@ function getModalPaddingStyles({
     modalContainerStylePaddingTop,
     modalContainerStylePaddingBottom,
     insets,
-}: ModalPaddingStylesArgs): ViewStyle | CSSProperties {
+}: ModalPaddingStylesParams): ViewStyle {
     // use fallback value for safeAreaPaddingBottom to keep padding bottom consistent with padding top.
     // More info: issue #17376
-    const safeAreaPaddingBottomWithFallback = insets.bottom === 0 ? modalContainerStylePaddingTop || 0 : safeAreaPaddingBottom;
+    const safeAreaPaddingBottomWithFallback = insets.bottom === 0 ? modalContainerStylePaddingTop ?? 0 : safeAreaPaddingBottom;
```

Comment:
why this change? `||` -> `??`

Reply:
I think the TS style guide doesn't specify whether Local OR Operator or Nullish Coalescing Operator should be used, but I remember someone is saying that ESLint rule forces the usage of Nullish Coalescing Operator.

Anyway, I just wanted to check because these are slightly different. But I gree nullsih coalescing operator is generally better if you're only accounting for `undefined` or `null`.

I just want to make sure that the existing code doesn't rely on the default value to be used when the left hand value is falsly value, such as `0` or `''`.

Reply:
> ESLint rule forces the usage of Nullish Coalescing Operator.

That's probably why it was changed 👍